src/evo/chainhelper.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2014-2024 The Dash Core developers
+// Distributed under the MIT/X11 software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_EVO_CHAINHELPER_H
+#define BITCOIN_EVO_CHAINHELPER_H
+
+#include <masternode/payments.h>
+
+class CChainstateHelper : public CMNPaymentsProcessor
```

Comment:
why there's used inheritance instead aggregation? Why do we even need CMNPaymentProcessor?